diff -up system-config-kdump-2.0.5/src/system-config-kdump.py.kernel_path system-config-kdump-2.0.5/src/system-config-kdump.py
--- system-config-kdump-2.0.5/src/system-config-kdump.py.kernel_path	2012-08-22 07:24:57.214957027 -0400
+++ system-config-kdump-2.0.5/src/system-config-kdump.py	2012-08-22 07:25:12.134926275 -0400
@@ -375,7 +375,7 @@ class MainWindow:
 
         self.total_mem = 0
         self.usable_mem = 0
-        self.kernel_prefix = "/"
+        self.kernel_prefix = ""
 
         self.running_kernel = os.popen("/bin/uname -r").read().strip()
 
@@ -1389,7 +1389,8 @@ class MainWindow:
                 cmd, kernel, error,
                 parent = self.toplevel)
         else:
-            self.kernel_prefix = kernel.rsplit("/", 1)[0]
+            if (self.arch != "s390x"):
+                self.kernel_prefix = kernel.rsplit("/", 1)[0]
             if DEBUG:
                 print "Default kernel = " + kernel
                 print "Kernel prefix = " + self.kernel_prefix
